src/themes/default/modules/dropdown.overrides

```diff
@@ -25,35 +25,35 @@
     width: auto;
 }
 .ui.dropdown > .dropdown.icon::before {
-    content: '\f0d7';
+    content: "\f0d7";
 }
 
 /* Sub Menu */
 .ui.dropdown .menu .item .dropdown.icon::before {
-    content: '\f0da'/*rtl:'\f0d9'*/;
```

Comment:
Don't remove the rtl hint

Reply:
fixed, related discussion https://github.com/fomantic/Fomantic-UI/pull/2612#discussion_r1050731482